packages/blocks/src/api/test/registration.js

```diff
@@ -637,4 +638,38 @@ describe( 'blocks', () => {
 			] );
 		} );
 	} );
+
+	describe( 'unregisterBlockStyle', () => {
+		afterEach( () => {
+			removeFilter( 'blocks.registerBlockType', 'my-plugin/block-with-styles/big/unregister' );
+			removeFilter( 'blocks.registerBlockType', 'my-plugin/block-with-styles/small/unregister' );
+		} );
+
+		it( 'should remove styles', () => {
+			unregisterBlockStyle( 'my-plugin/block-with-styles', 'big' );
+			const settings = registerBlockType( 'my-plugin/block-with-styles', {
```

Comment:
There should maybe be a test where we verify removal of a prior `registerBlockStyle`.